src/main/java/duke/Event.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected String timeQualifier;
+    protected LocalDate timeDescription;
+    protected String timeDescriptionInput;
+
+    Event(String description, String timeQualifier, String timeDescription) {
+        super(description);
+        this.timeQualifier = timeQualifier;
+        this.timeDescription = LocalDate.parse(timeDescription);
+        this.timeDescriptionInput = timeDescription;
+    }
+
+    @Override
+    public String getTaskTypeIcon() {
+        return "E";
+    }
+
+    @Override
+    public String toWrite() {
+        return getTaskTypeIcon() + "," + (super.isDone ? "1," : "0,") + super.description + "," + timeQualifier + "," + timeDescriptionInput + "\n";
```

Comment:
Since you are using quite a long String concatenation here, would String.format() make things clearer?